market.cpp

```diff
@@ -28,26 +28,7 @@
 #include "gameio.h"
 #include "gamedata.h"
 
-Market::Market()
```

Comment:
I moved these into the header file with the more modern style of member variable initialization.